slugify/__main__.py

```diff
@@ -39,13 +39,13 @@ def parse_args(argv):
     parser.add_argument("--allow-unicode", action='store_true', default=False,
                         help="Allow unicode characters")
 
-    args = parser.parse_args(argv[1:])
+    args = parser.parse_args(argv[1:] if argv else [])
```

Comment:
argv cannot be None so we should not need this logic. If mypy cannot understand what is happening in line 86, we should find instead ways of telling it. This means the type for argv in line 8 should not be an optional list but a `list[str]`